internal/server/api.go

```diff
@@ -118,22 +118,19 @@ func (a *API) ServeHTTP(_ http.ResponseWriter, r *http.Request) {
 		a.pushError(err)
 	}
 
+	if kind == "increment_metric" {
+		// Let's just output the metrics data and stop
+		a.outputRequestData(kind, actual)
+		return
+	}
+
 	if err := a.pushResult(kind, actual); err != nil {
```

Comment:
Oh you might need to skip the increment_metric here or it adds it to the assertions I think?

Reply:
Ah, does the return on line 24 not break us out of the method?

Reply:
Oh yes, I see you extracted the stdout output, perfect! 👍